core/helpers/causeDebt.ts

```diff
@@ -0,0 +1,118 @@
+import getContract, { getContractAddressByName } from '../src/contracts';
+import BigNumber from '../src/bignumber';
+import { WAD_NUMBER_OF_DIGITS } from '../src/constants/UNITS';
+import { EthereumProvider } from 'hardhat/types';
+import { ethers } from 'ethers';
+import { formatToHex } from './format';
+
+export const causeDebt = async (
+    network: string,
+    provider: EthereumProvider,
+    debtAmountDai: BigNumber,
+    mkrOnAuction: BigNumber,
+    daiOnAuction: BigNumber
+) => {
+    await overwriteQueuedDebt(network, provider, debtAmountDai);
+    await ensureQueuedDebtEqual(network, debtAmountDai);
+    await overwriteProtocolOwnDaiBalance(network, provider);
+    await ensureDaiIsZero(network);
+    await overwriteLottedMkrAmount(network, provider, mkrOnAuction);
+    await ensureLottedMkrHasValue(network, mkrOnAuction);
+    await overwriteBidDaiAmount(network, provider, daiOnAuction);
+    await ensureBidDaiHasValue(network, daiOnAuction);
+    await startDebtAuction(network);
+};
+
+const overwriteValueInSlot = async (
+    network: string,
+    contractName: string,
+    slotAddress: string,
+    provider: EthereumProvider,
+    newValue: string
```

Comment:
Please keep arguments in the same order if possible, meaning:
```suggestion
    network: string,
    provider: EthereumProvider,
    contractName: string,
    slotAddress: string,
    newValue: string
```